salt/states/file.py

```diff
@@ -3056,6 +3056,7 @@ def managed(
 
             if isinstance(ret["changes"], tuple):
```

Comment:
`file.check_managed_changes` usually returns ret dict from `file.check_file_meta`, unless `file.get_managed` has comments about failure, [in which case it returns a tuple](https://github.com/saltstack/salt/blob/1ebaed1ff0e1ccf5fb67050de19ce642ab8d28d2/salt/modules/file.py#L5416-L5418) `(False, comment)`. This previously only partially accounted for that, needs to reset changes to empty dict to avoid complaint about malformed state return.